examples/typescript-enum-type/index.spec.ts

```diff
@@ -9,15 +9,15 @@ describe('Should be able to render correct enums based on options', () => {
   });
   test('and should log expected output to console', async function testIt(): Promise<void> {
     // Wait for first call to the `generate` function is over, and we can reliably know the output
-    if (spy.mock.calls.length !== 6) {
+    if (spy.mock.calls.length !== 3) {
       await timeout(50);
       return testIt();
     }
```

Comment:
```suggestion
```

You can safely remove this 🙂

Reply:
i have removed this piece of code but still i'm getting failed test for this spec file. can you help me ?
Thanks

Reply:
Have you updated the snapshots? `npm run test -- -u`

Reply:
yes but on snapshot creation it gives error.

Reply:
Which error are you getting?

Reply:
im getting below error 
`FAIL examples/typescript-enum-type/index.spec.ts (18.144 s)
  ● Should be able to render correct enums based on options › and should log expected output to console

    expect(received).toMatchSnapshot()

    Snapshot name: `Should be able to render correct enums based on options and should log expected output to console 1`

    - Snapshot  - 14
    + Received  +  0

    @@ -1,19 +1,5 @@
    - Generator output with Union:
    - class Root {
    -   private _eventType?: EventType;
    -
    -   constructor(input: {
    -     eventType?: EventType,
    -   }) {
    -     this._eventType = input.eventType;
    -   }
    -
    -   get eventType(): EventType | undefined { return this._eventType; }
    -   set eventType(eventType: EventType | undefined) { this._eventType = eventType; }
    - }
    - type EventType = "ping" | "pong";
      Generator output with Enum:
      class Root {
        private _eventType?: EventType;

        constructor(input: {

      12 |     expect(spy.mock.calls.length).toEqual(6);
      13 |     const generatedContent = spy.mock.calls.slice(3).join('\n');
    > 14 |     expect(generatedContent).toMatchSnapshot();
         |                              ^
      15 |   });
      16 | });
      17 |

      at examples/typescript-enum-type/index.spec.ts:14:30
      at fulfilled (examples/typescript-enum-type/index.spec.ts:5:58)
`

Reply:
please see https://github.com/asyncapi/modelina/pull/868#pullrequestreview-1117781992